CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ project(Atlas CXX)
 set (SIM_BASE ${PROJECT_SOURCE_DIR})
 
 # Profile build flags
-set(CMAKE_CXX_FLAGS_FASTDEBUG "-O3 -g")
+set(CMAKE_CXX_FLAGS_FASTDEBUG "-O3 -g -DNDEBUG")
```

Comment:
This bit me again.  My latest conda env toggles this value between some of the CMakeLists.  Dunno why.  Force the value.